CHANGES.md

```diff
@@ -8,6 +8,8 @@
 
 ### Changed
 
+- Abstracted Sig_item_map into is own module (#98, @azzsal)
```

Comment:
This is internal so we don't need a changelog entry for it, I simply forgot to add the `no-changelog` label. You can safely remove it!